app/controllers/new-signup.js

```diff
@@ -0,0 +1,68 @@
+import Controller from '@ember/controller';
+import { action, set } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import registerUser from '../utils/register-api';
+
+export default class SignupController extends Controller {
+  queryParams = ['state', 'dev'];
```

Comment:
Can we use the Router Service here please, instead of queryParams?

1. When the queryParams disappear from Controller in future, it will make our job easier.
2. Easier to mock injected service and perform testing

Reply:
https://github.com/Real-Dev-Squad/website-my/issues/247